understanding/labels-or-instructions.html

```diff
@@ -58,182 +56,119 @@
                   
                </p>
-               <p>The intent of this Success Criterion is not to clutter the page with unnecessary information
-                  but to provide important cues and instructions that will benefit people with disabilities.
-                  Too much information or instruction can be just as harmful as too little.
-                  The goal is to make certain that enough information is provided for the user to accomplish
-                  the task without undue confusion or navigation.
+               <p>この達成基準の意図は、ページを不要な情報だらけにしてしまうことではなく、障害のある利用者に役立つ重要な手がかり及び説明文を提供することである。情報又は指示が多すぎると、少なすぎる場合と同じくらい有害になることもある。目標は、利用者の混乱を最小限に抑えて、必要以上のナビゲーションを提供することなく、利用者がタスクを完了するために十分な情報を提供することである。
                   
                </p>
-               <p>This Success Criterion does not require that labels or instructions be correctly marked
-                  up, 
-                  identified, or associated with their respective controls - this aspect is covered
-                  separately by
-                  <a href="info-and-relationships">1.3.1: Info and Relationships</a>. It is possible for content
-                  to pass this Success Criterion (providing relevant labels and instructions) while
-                  failing
-                  Success Criterion 1.3.1 (if the labels or instructions aren't correctly marked up,
-                  identified, or associated).
+               <p>この達成基準では、ラベル又は説明が正確にマークアップされる、識別される、又はそれぞれのコントロールに関連付けられることを要求しているわけではない。この観点は、<a href="info-and-relationships">1.3.1: 情報及び関係性</a>で個別に扱われる。なお、コンテンツが達成基準 1.3.1 に失敗している (ラベル又は説明が正しくマークアップ、識別、又は関連付けられていない) 場合でも、この達成基準には合格している (関連するラベル及び説明が提供されている) 場合がある。
                   
                </p>
-               <p>Further, this Success Criterion does not take into consideration whether or not alternative
-                  methods of
-                  providing an accessible name or description for form controls and inputs has been
-                  used - this aspect is
-                  covered separately by <a href="name-role-value">4.1.2: Name, Role and Value</a>. It is possible
-                  for controls and inputs to have an appropriate accessible name or description (e.g.
-                  using <code>aria-label="..."</code>)
-                  and therefore pass Success Criterion 4.1.2, but to still fail this Success Criterion
-                  (if the labels or instructions
-                  aren't presented to all users, not just those using assistive technologies).
-                  
+               <p>さらに、この達成基準では、フォームのコントロール及び入力欄にアクセシブルな名前 (accessible name) 又は説明を提供するための代替方法が使用されているかどうかは考慮されない。この観点は、<a href="name-role-value">4.1.2: 名前 (name)・役割 (role)・値 (value)</a> で個別に扱われる。コントロール及び入力欄が適切なアクセシブルな名前 (accessible name) 又は説明 (例えば、<code>aria-label="..."</code> を使用する) を持ち、よって達成基準 4.1.2 に合格することが可能であるが、この達成基準には失敗する (支援技術を使用しているユーザーだけでなく、すべてのユーザーにラベル又は指示が提示されていない) 場合がある。
```

Comment:
最後の文は微妙に誤訳のような?
特に最後の括弧内、原文には aren't ... not just という二重否定があり、
「『支援技術を使用しているユーザーだけでなく (not just) すべてのユーザーに提示されている』、というわけではない (aren't)」
という意味なんですが、いまの訳だと「すべてのユーザーにラベル又は指示が提示されていない」と読めて誤解を招くように思います。
こんな感じにしてはどうでしょうか。

```suggestion
               <p>さらに、この達成基準では、フォームのコントロール及び入力欄にアクセシブルな名前 (accessible name) 又は説明を提供するための代替方法が使用されているかどうかは考慮されない。この観点は、<a href="name-role-value">4.1.2: 名前 (name)・役割 (role)・値 (value)</a> で個別に扱われる。コントロール及び入力欄が適切なアクセシブルな名前 (accessible name) 又は説明を (例えば <code>aria-label="..."</code> を使用して) 持っており、これによって達成基準 4.1.2 を満たすが、この達成基準には失敗するということもありえる (ラベル又は説明がすべてのユーザーに提示されているのではなく、支援技術を使用しているユーザーだけに提示されている場合) 。
```